src/backend/parser/gram.y

```diff
@@ -6046,6 +6046,15 @@ DistributedBy:   DISTRIBUTED BY  '(' distributed_by_list ')'
 				distributedBy->keyCols = NIL;
 				$$ = (Node *)distributedBy;
 			}
+			| DISTRIBUTED LOCAL
+			{
+				DistributedBy *distributedBy = makeNode(DistributedBy);
+				distributedBy->ptype = POLICYTYPE_LOCAL;
```

Comment:
we did choose POLICYTYPE_LOCAL Instead of POLICYTYPE_ENRTY, because `getPolicyForDistributedBy` explicitly forbits creating relation with POLICYTYPE_ENRTY policy. We are unaware of reason however.

Reply:
Possible we keep distributeBy as NULL like pg_class? So that planner could work properly

Reply:
Will think about that